exonum/src/encoding/tests.rs

```diff
@@ -24,22 +24,51 @@ use messages::{RawMessage, Message, FromRaw, Connect, Propose, Prevote, Precommi
 
 use super::{Field, Offset};
 
+#[allow(dead_code)]
+mod ignore_new {
```

Comment:
I use `encoding_struct` to set buffer schema. So I need to ignore `dead_code` warning about `new` method. I add some comment.